frontend/resourceadm/components/ResourcePageInputs/ResourceLanguageTextField.tsx

```diff
@@ -143,6 +221,17 @@ export const ResourceLanguageTextField = ({
   );
 };
 
+interface MarkdownToolbarButtonProps {
```

Comment:
Can see that `type` was used instead of `interface` at the top of the file, maybe we should keep it consistent? 🤔